scripts/cli-wrapper-gen.py

```diff
@@ -131,7 +141,7 @@ def nargs(item):
             for subcommand in command["subcommands"]:
                 if "arguments" in subcommand:
                     for argument in subcommand["arguments"]:
-                        argument["required"] = False if "default" not in argument else True
+                        argument["required"] = argument["required"] if 'required' in argument else ("default" in argument)
```

Comment:
Shouldn't the latter part be `"default" not in argument` to make is optional if it has a default? 🧐